external-crates/move/crates/move-package/src/resolution/dependency_graph.rs

```diff
@@ -240,13 +250,25 @@ impl<Progress: Write> DependencyGraphBuilder<Progress> {
             })
             .unwrap_or(None);
 
+        // implicits deps should be skipped if the manifest contains any of them
+        // explicitly (or if the manifest is for a system package).
+        let add_implicits: bool = !self.implicit_deps.iter().any(|(name, _)| {
+            root_manifest.dependencies.contains_key(name) || root_manifest.package.name == *name
```

Comment:
Can we require the `override = true` be set here? May help us out in the future possibly (and also makes it clear that an override is happening here).